addons/binding/org.openhab.binding.avmfritz/ESH-INF/i18n/avmfritz_de.properties

```diff
@@ -83,30 +72,6 @@ bridge-type.config.avmfritz.fritzpowerline.asyncTimeout.description = Timeout f
 bridge-type.config.avmfritz.fritzpowerline.syncTimeout.label = Synchroner Timeout
 bridge-type.config.avmfritz.fritzpowerline.syncTimeout.description = Timeout f�r synchrone Abfragen des FRITZ!Powerline Ger�t (in Millisekunden).
 
-bridge-type.config.avmfritz.FRITZ_Powerline_546E_Solo.ipAddress.label = IP-Adresse
-bridge-type.config.avmfritz.FRITZ_Powerline_546E_Solo.ipAddress.description = Lokale IP-Adresse oder Hostname des FRITZ!Powerline 546E Ger�tes.
-
-bridge-type.config.avmfritz.FRITZ_Powerline_546E_Solo.port.label = Port
```

Comment:
Yes, because the FRITZ_Powerline_546E_Solo isn't a thing-type anymore. It's now a brige-type with a related config-description-uri (fritzpowerline). Removing duplicate here.